docs/data-cookbooks/cnm-workflow.md

```diff
@@ -469,15 +473,15 @@ The data written to the `CnmResponseStream` should adhere to the [Response Messa
   "identifier": "1234-abcd-efg0-9876",
   "response": {
     "status": "FAILURE",
-    "errorCode": "INGEST_ERROR",
+    "errorCode": "PROCESSING_ERROR",
     "errorMessage": "File [cumulus-dev-a4d38f59-5e57-590c-a2be-58640db02d91/prod_20170926T11:30:36/production_file.nc] did not match gve checksum value."
   }
 }
 ```
 
 Note the `CnmResponse` state defined in the `.tf` workflow definition above configures `$.exception` to be passed to the `CnmResponse` Lambda keyed under `config.WorkflowException`. This is required for the `CnmResponse` code to deliver a failure response.
 
-To test the failure scenario, send a record missing the `collection` key.
+To test the failure scenario, send a record missing the `product.name` key.
```

Comment:
Sending a record without a `collection` key would just cause the `messageConsumer` to never start the workflow, since that is part of its schema validation